.github/workflows/integrate.yaml

```
@@ -98,7 +98,7 @@ jobs:
           provider: microk8s
           channel: 1.29-strict/stable
           juju-channel: 3.4/stable
-          charmcraft-channel: latest/candidate
+          charmcraft-channel: 3.1/stable
```

Please add a comment that says why we are pinning this version and when we should unpin, for example:

```suggestion
          # Pinned to 3.1/stable due to https://github.com/canonical/charmcraft/issues/1845
          charmcraft-channel: 3.1/stable
```

---

Good practice, thanks!